ActionLogger/src/pl/jtat/logger/C.java

```diff
@@ -0,0 +1,27 @@
+package pl.jtat.logger;
```

Comment:
Raczej nie będziemy się trzymać  takiej konwencji  w całym projekcie i nie bierzemy pod uwage w ogóle różnych wersji językowych. Najlepiej jakby tej klasy w ogóle nie było, a jak była to jakaś  internal, tylko dla Waszego modułu.

Reply:
Różne wersje językowe logów :')?